comp/core/tagger/taggerimpl/collectors/workloadmeta_extract.go

```diff
@@ -199,6 +199,7 @@ func (c *WorkloadMetaCollector) handleContainer(ev workloadmeta.Event) []*types.
 	tagList.AddLow(tags.ShortImage, image.ShortName)
 	tagList.AddLow(tags.ImageTag, image.Tag)
 	tagList.AddLow(tags.ImageID, image.ID)
+	tagList.AddLow(tags.KubeGPUType, container.Resources.GPUType)
```

Comment:
Can you add a test for this tag in `comp/core/tagger/taggerimpl/collectors/workloadmeta_test.go`?
I think we should add a new case or update one of the existing ones in `func TestHandleContainer`.